src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const a = numberOne ? Big(numberOne) : null;
+  const b = numberTwo ? Big(numberTwo) : null;
+  let result = 0;
+
+  if (operation === '+') {
+    result = Big(a.plus(b));
+  } else if (operation === '-') {
+    result = Big(a.minus(b));
+  } else if (operation === 'x') {
+    result = Big(a.times(b));
+  } else if (operation === '÷') {
+    try {
+      result = Big(a.div(b));
+    } catch (e) {
+      return "Can't be divided by 0";
+    }
+  } else if (operation === '%') {
+    result = Big(a.div(100));
+  }
```

Comment:
Let us fix the "%" operation. Instead of `result = Big(a.div(100)` we can write `result = a.mod(b)`. Where the "a" and "b" are declared with the Big(numberOne) and Big(numberTwo) already